packages/node_modules/nav-frontend-lesmerpanel/src/lesmerpanel.tsx

```diff
@@ -0,0 +1,264 @@
+                    this.mer.focus();
+                }
+            } else {
+                this.setState({
+                    hindreToggle: false,
+                    visInnhold: false
+                });
+                if (!erSynligIViewport(this.lesMerPanel)) {
+                    scrollTo(this.lesMerPanel, 600);
+                }
+            }
+        }
+    }
+
+    setAutoHoyde() {
+        /* Fjerner animasjonsklassen slik at Safari ikke
+        tegner komponenten på nytt når høyde settes til 'auto': */
+        this.setState({
+            containerClassName: ''
+        });
+        // Setter høyde til auto:
+        setTimeout(
+            () => {
+                this.setState({
+                    hoyde: 'auto',
+                    containerClassName: ''
+                });
+            },
+            0);
+    }
+
+    apne() {
+        this.setState({
+            hoyde: '0',
+            hindreToggle: true,
+            containerClassName: ' lesMerPanel__merContainer--medAnimasjon',
+            visInnhold: true,
+            harTransisjon: true
+        });
+        setTimeout(
+            () => {
+                const hoyde = this.mer ? this.mer.offsetHeight : '0';
+                this.setState({
+                    hoyde,
+                    erApen: true
+                });
+            },
+            10);
+    }
+
+    lukk() {
+        const hoyde = this.mer ? this.mer.offsetHeight : '0';
+        this.setState({
+            hoyde,
+            hindreToggle: true,
+            harTransisjon: true
+        });
+        setTimeout(
+            () => {
+                this.setState({
+                    containerClassName: ' lesMerPanel__merContainer--medAnimasjon',
+                    hoyde: '0',
+                    erApen: false
+                });
+            },
+            10);
+    }
+
+    toggle(e, onOpen, onClose) {
+        e.preventDefault();
+        if (!this.state.hindreToggle) {
+            /* hindreToggle for å hindre dobbelklikk,
+            eller at noen klikker mens animasjonen pågår.
+            Dobbelklikk vil skape kluss med logikken. */
+            if (this.state.erApen) {
+                this.lukk();
+                onClose();
+            } else {
+                this.apne();
+                onOpen();
+            }
+        }
+    }
+
+    render() {
+        const { className, intro, id, mer, onOpen, onClose } = this.props;
+        return (
+            <div
+                id={id}
+                ref={(c) => {
+                    this.lesMerPanel = c;
+                }}
+                className={`lesMerPanel ${className}`}
+            >
+                <div>
+                    {intro}
+                </div>
+                <div
+                    style={{ height: this.state.hoyde }}
+                    className={`lesMerPanel__merContainer${this.state.containerClassName}`}
+                    onTransitionEnd={() => {
```

Comment:
`onTransitionEnd={this.onTransitionEnd}` ?

Reply:
Ok, men da må den vel også bindes i constructor?

```
this.onTransitionEnd = this.onTransitionEnd.bind(this);
```